core/trino-main/src/main/java/io/trino/sql/analyzer/FeaturesConfig.java

```diff
@@ -374,7 +374,7 @@ public FeaturesConfig setUsePreferredWritePartitioning(boolean usePreferredWrite
         return this;
     }
 
-    @Min(0)
+    @Min(1)
```

Comment:
actually I think it's wrong since partition count estimate can be lower than `1` (double). See `io.trino.sql.planner.iterative.rule.DeterminePreferredWritePartitioning#apply` and `io.trino.cost.AggregationStatsRule#getRowsCount`.